app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -63,8 +76,26 @@ object Project extends GeoJsonSupport {
         description,
         visibility,
         tileVisibility,
+        isAOIProject,
+        aoiCadenceMillis,
+        new Timestamp(now.getTime - aoiCadenceMillis),
         tags
       )
     }
   }
+
+  object Create {
+    /** Custon Circe decoder for [[Create]], to handle default values. */
+    implicit val projectDecoder: Decoder[Create] = Decoder.instance(c =>
+      (c.downField("organizationId").as[UUID]
+        |@| c.downField("name").as[String]
```

Comment:
I haven't seen `|@|` before -- so where you're using it, the results of the functions (`c.downField`, etc) are passed as a `tuple` to the `Create.apply` call?